src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -13402,6 +13402,41 @@ class C
                 Diagnostic(ErrorCode.ERR_ParameterIsStaticClass, "set").WithArguments("S").WithLocation(4, 11));
         }
 
+        [WorkItem(63975, "https://github.com/dotnet/roslyn/pull/63975")]
```

Comment:
>pull [](http://example.com/codeflow?start=59&length=4)

The WorkItem attribute is supposed to reference an issue rather than a pull request

Reply:
Good catch, I copied the wrong link. Fixed: eda9c839e76b934cad9943fac8d494c33ccbf2bf